src/styles/layout/blog/banner.scss

```diff
@@ -51,6 +54,68 @@
   }
 
   &__link--selected {
-    color: rgba(231, 238, 247, 0.66);
+    color: rgb(231 238 247 / 0.66);
+  }
+}
+
+@include mixins.tablet {
+  .banner {
+    &__container-blog {
+      padding-top: 24px;
+      padding-bottom: 84px;
+      min-height: 0;
+    }
+
+    &__title {
+      font-size: 36px;
+      line-height: 48px;
+    }
+  }
+}
+
+@include mixins.phone {
+  .banner {
+    &__container-blog {
```

Comment:
Please check styles which repeated on the different resolution. And check it throw dev tools and inspector